Getting-Started-with-Akka-http-signature.md

```diff
@@ -114,111 +114,77 @@ This will save the contents of the key in String format within the file found by
+hHhTWQIDAQAB"""
 
-@ import Cert._
 ```
 
-After this all dependencies should be resolved and the user will be able to transform the keys into a Pointed Graph:
+After this all dependencies should be resolved and the user will be able to transform the keys into a Pointed Graph using the following function
 
 ```scala
-@ val keyPG = pub.toPG
+@  def pubKeyPG = {	
+      import ops._
+      implicit val bind = run.cosy.crypto.Cert.binderWithName[Rdf](Uri("#key"))
+      pubKey.map(_.toPG)
+   } 
 
-keyPG : PointedGraph[RDF] = org.w3.banana.PointedGraph$$anon$1@7f8fa63d
+defined function pubKeyPG
 ```
-One can then retrieve both the pointer and the graph:
+After that the user can simply call said function to get a pointed graph:
 
 ```scala
-@ val keyGraph = keyGraph.graph
+@ val keyGraph = pubKeyPG
 
-keyGraph : RDF#Graph =  {5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0 http://www.w3.org/ns/auth/cert#exponent "65537"^^http://www.w3.org/2001/XMLSchema#integer; 5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0 @http://www.w3.org/ns/auth/cert#modulus "00937adccd722bc982aed4847872b81e36b890bca13166714bc2befe4d8547b6218ecd2da1eb020198a4ea00e4db6757c7dda738ec8db8b3bf211d3a3a17e196a2035bc4c79d06d8a581487d9f49e86374712b10ef500dfa242a20cab52911e2636c9d99b21fe9768ef2381989a25dc8b0b7a46531249aac27c4b8ab451a19d5fbdfa5f78b0deac9778c7ff87cf6106ae4a6433466beb21df1265bf1fc9ab9cc80d7aff8cc4d0f67ae28647e4048da8df753493b8de6a8e0961416b4b37f7012907d2e756034b8a84c6e495c8f1d81f69843ae51379571d83b38e1c08a08c3748ef75ed7ecb016d0c8426b30c8c5060a08f87f6764b0ec14667cd6f0daa1244157"^^http://www.w3.org/2001/XMLSchema#hexBinary}
-
-@ val keyRDFPointer = keyGraph.pointer
-
-keyRDFPointer: RDF#Node = 5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0
-```
-As is evident, the pointer is an automatically generated Blank Node, which can be quite difficult to process. Because of this, it would be more optimal to change the pointer to not be a blank node but rather a #uri. The user can change the pointer by using the following function : 
-
-```scala
-@ implicit class PGwrapper(val pg: PointedGraph[Rdf]) extends AnyVal {
-  def rename(to: Rdf#Node)(implicit ops: RDFOps[Rdf]): PointedGraph[Rdf] = {
-    val oldNode = pg.pointer
-    PointedGraph[Rdf](
-      to,
-      ops.makeGraph(pg.graph.triples.map{ triple =>
-        ops.fromTriple(triple) match {
-          case (oldNode,rel,obj) => ops.makeTriple(to,rel,obj)
-          case (subj,rel,oldNode) => ops.makeTriple(subj,rel,to)
-          case _ => triple
-        }
-      })
-    )
-  }
-}
-
-defined class PGwrapper
+res44: Try[PointedGraph[Jena]] = Success(org.w3.banana.PointedGraph$$anon$1@5487b93e)
 ```
-
-After this, one can simply change the header by into something simple by calling this function like so:
+The function pubKeyPG returns a Try[PointedGraph[Jena]]. We can then retreive both the pointer and the graph from this PointedGraph like so: 
```

Comment:
you should be able to do that in one step I think using `val (pointer,graph) = ...`